main.go

```diff
@@ -40,118 +42,205 @@ import (
 )
 
 func main() {
+	Logger.Info("Starting Nancy")
+
 	if len(os.Args) > 1 && os.Args[1] == "iq" {
+		Logger.Info("Nancy parsing config for IQ")
 		config, err := configuration.ParseIQ(os.Args[2:])
 		if err != nil {
 			flag.Usage()
 			os.Exit(1)
 		}
+		Logger.WithField("config", config).Info("Obtained IQ config")
 		processIQConfig(config)
+		Logger.Info("Nancy finished parsing config for IQ")
 	} else {
+		Logger.Info("Nancy parsing config for OSS Index")
 		ossIndexConfig, err := configuration.Parse(os.Args[1:])
 		if err != nil {
 			flag.Usage()
 			os.Exit(1)
 		}
 		processConfig(ossIndexConfig)
+		Logger.Info("Nancy finished parsing config for OSS Index")
 	}
 }
 
 func printHeader(print bool) {
 	if print {
+		Logger.Info("Attempting to print header")
 		figure.NewFigure("Nancy", "larry3d", true).Print()
 		figure.NewFigure("By Sonatype & Friends", "pepper", true).Print()
+
+		Logger.WithField("version", buildversion.BuildVersion).Info("Printing Nancy version")
 		log.Println("Nancy version: " + buildversion.BuildVersion)
```

Comment:
Down further when config.Version is set we log all the info around version. Do we want to do that here too?

Reply:
Honestly we probably don't need the `log.Println` here now, since I limit this whole section to being printed if we have `quiet`. WDYT?

Reply:
id we added it as part of #52 so i believe removing it will basically revert that change.

Reply:
What I meant is now:

```
	if print {
		LogLady.Info("Attempting to print header")
		figure.NewFigure("Nancy", "larry3d", true).Print()
		figure.NewFigure("By Sonatype & Friends", "pepper", true).Print()

		LogLady.WithField("version", buildversion.BuildVersion).Info("Printing Nancy version")
		log.Println("Nancy version: " + buildversion.BuildVersion)
		LogLady.Info("Finished printing header")
	}
```

It's surrounded in `print`, which I think means we can switch the l`og.Println` to `fmt.Println`

Reply:
Yeah and looking a little more i think we can actually remove this whole chunk now. 

``` 
if config.Quiet {
		LogLady.Debug("Setting console log output to discard, quiet requested")
		log.SetOutput(ioutil.Discard)
	}
```

Basically if we need to log something "tracing" wise we will use `LogLady` and if it needs to go to the console it should probably be baked into the logrus formatters we have now that spit out Json, Text, Csv, etc. I'm thinking it might make sense to bake the header output into the auditlog.go then we will have all the "console" output in one place. But i digress. 

If youre cool with it flip that to a fmt.Println and wipe that if check. I'll probably create an issue to clean up the header stuff and move it.

Reply:
Will do!